include/exadg/convection_diffusion/driver.cpp

```diff
@@ -58,16 +58,17 @@ Driver<dim, Number>::setup()
 
   pcout << std::endl << "Setting up scalar convection-diffusion solver:" << std::endl;
 
-  application->setup();
+  grid = std::make_shared<Grid<dim>>();
```

Comment:
I think this is an "inconsistency" in the sense that `mapping` is created inside `Application::setup()` (or https://github.com/exadg/exadg/blob/67e0beaa957e174a8b4fb526fdb703ba4aa4331f/include/exadg/grid/grid_utilities.h#L52), while `grid` is created outside. I would prefer to do it uniformly.


I think it would be in line with other code in ExaDG that `create()`-functions also create the object (i.e. call the constructor). Hence, this line should probably be shifted inside the `setup()` function.